x-pack/plugins/security_solution/cypress/e2e/exceptions/alerts_table_flow/endpoint_exceptions.cy.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  goToClosedAlertsOnRuleDetailsPage,
+  openAddEndpointExceptionFromFirstAlert,
+} from '../../../tasks/alerts';
+import { deleteAlertsAndRules } from '../../../tasks/common';
+import { login, visitWithoutDateRange } from '../../../tasks/login';
+import { getEndpointRule } from '../../../objects/rule';
+import { goToRuleDetails } from '../../../tasks/alerts_detection_rules';
+import { createRule } from '../../../tasks/api_calls/rules';
+import {
+  waitForAlertsToPopulate,
+  waitForTheRuleToBeExecuted,
+} from '../../../tasks/create_new_rule';
+import {
+  esArchiverLoad,
+  esArchiverResetKibana,
+  esArchiverUnload,
+} from '../../../tasks/es_archiver';
+import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../../urls/navigation';
+import {
+  addExceptionFlyoutItemName,
+  selectCloseSingleAlerts,
+  submitNewExceptionItem,
+} from '../../../tasks/exceptions';
+import { ALERTS_COUNT, EMPTY_ALERT_TABLE } from '../../../screens/alerts';
+import {
+  EXCEPTION_ITEM_CONTAINER,
+  FIELD_INPUT_PARENT,
+  NO_EXCEPTIONS_EXIST_PROMPT,
+} from '../../../screens/exceptions';
+import {
+  removeException,
+  goToAlertsTab,
+  goToEndpointExceptionsTab,
+} from '../../../tasks/rule_details';
+
+describe('Endpoint Exceptions workflows from Alert', () => {
+  const expectedNumberOfAlerts = 1;
+  before(() => {
+    esArchiverResetKibana();
+    esArchiverLoad('endpoint');
+    login();
+    deleteAlertsAndRules();
+    createRule(getEndpointRule());
+  });
+  beforeEach(() => {
+    visitWithoutDateRange(DETECTIONS_RULE_MANAGEMENT_URL);
+    goToRuleDetails();
+    waitForTheRuleToBeExecuted();
+    waitForAlertsToPopulate();
+  });
+  after(() => {
+    esArchiverUnload('endpoint');
+  });
+
+  it('Should be able to create and close single Endpoint exception from overflow menu', () => {
+    // The Endpoint will populated with predefined fields
+    openAddEndpointExceptionFromFirstAlert();
+
+    // As the endpoint.alerts-* is used to trigger the alert the
+    // file.Ext.code_signature will be populated as the first item
+    cy.get(EXCEPTION_ITEM_CONTAINER)
+      .eq(0)
+      .find(FIELD_INPUT_PARENT)
+      .eq(0)
+      .should('have.text', 'file.Ext.code_signature');
+
+    // The Endpoint will populated with predefined fields
+    openAddEndpointExceptionFromFirstAlert();
```

Comment:
Is there any reason why we are opening the first alert twice?

Reply:
Yes, but I am not sure why we face this experience the flyout the first time closes before the user enters the exception details, that's why we need to open it twice until we fix the flyout reloading

Reply:
The test is quite flaky :( I will leave it to the end, if I wasn't able to fix then I will skip until fixing the flyout